README.md

```diff
@@ -2,7 +2,7 @@
 
 ## What is this repository? <a name="what_is_it"></a>
 This repository contains tools, templates, and information for assembling, debugging, testing, 
-and running your custom inference models and custom tasks with DataRobot.
+and running your custom inference models, custom tasks and custom notebook environments with DataRobot.
 
```

Comment:
Add a sentence to this section:
The [./public_dropin_notebook_environments](public_dropin_notebook_environments/) contains template examples (sample Dockerfile and context) for how to create custom images to use as the environments for DataRobot Notebooks.